test/integration/kic_custom_network_test.go

```diff
@@ -0,0 +1,58 @@
+// +build integration
+
+/*
+Copyright 2020 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"context"
+	"os/exec"
+	"testing"
+)
+
+func TestKicCustomNetwork(t *testing.T) {
+	if !KicDriver() {
+		t.Skip("only runs with docker driver")
+	}
+
+	tests := []struct {
+		description string
+		flag        string
+	}{
+		{
+			description: "create custom network",
```

Comment:
this doesn't seem to add any value to have just a "standard start" since we have that covered in other tests

Reply:
consider adding a network name other than bridge

for example "custom-network-1234"